README.md

```diff
@@ -54,6 +54,8 @@ The ps library is compatible with all AIX format descriptors of the ps command-l
   - Set of inheritable capabilities. See capabilities(7) for more information.
 - **capprm**
   - Set of permitted capabilities. See capabilities(7) for more information.
+- **capamb**
```

Comment:
Can you move this a few entries up to keep the sorted list?

Reply:
I see you added a new commit, but the entry didn't move up. Maybe you missed `git add`?

Reply:
oops, my bad